pkg/services/provisioning/provisioning.go

```diff
@@ -18,34 +18,43 @@ import (
 )
 
 type ProvisioningService interface {
+	RunInitProvisioners() error
 	ProvisionDatasources() error
 	ProvisionPlugins() error
 	ProvisionNotifications() error
 	ProvisionDashboards() error
 	GetDashboardProvisionerResolvedPath(name string) string
 	GetAllowUIUpdatesFromConfig(name string) bool
+	registry.BackgroundService
```

Comment:
Is it ok to add this to the interface since I need it in the override. (I can always type assert to registry.BackgroundService in the override but this is cleaner IMO)

Reply:
Probably fine?

Reply:
Thanks :pray:

Reply:
Nit, but would you mind moving this to line 21, i.e. at the top of the `ProvisioningService` body? I think it's more conventional to put embedded types at the beginning of the type body, since they're easier to spot.

Reply:
Ok will do right now